jvm/src/test/scala/tastyquery/BaseUnpicklingSuite.scala

```diff
@@ -54,16 +54,19 @@ abstract class BaseUnpicklingSuite(withClasses: Boolean, withStdLib: Boolean, al
     (base, classRoot)
   }
 
-  def resolve(path: DeclarationPath)(implicit ctx: BaseContext): Symbol =
-    followPath(defn.RootPackage, path)
+  /* TODO Currently this only resolves symbols that have already been loaded,
```

Comment:
Which test prompted this comment?

Reply:
Oh this comment should be removed in the second commit, actually. I removed it now.